fix: corrected tracepoints descriptions (#345)
tier4/caret_doc: docs/recording/trace_filtering.md

# /parameter_events:\
# "
```

<prettier-ignore-start>
!!! Info

>         If the node cannot be identified from the trace point information, the trace is collected as not applicable to the filter.

Reviewer: I made an alternative. This wording may be easier for users to understand.

```suggestion
        If the node cannot be identified from the trace point information, the trace will be collected without applying the filter.
```
